sql-select/actor-forty.sql

```diff
@@ -0,0 +1,3 @@
+select *
+  from "actors"
+  where "actorId" = '40';
```

Comment:
Numbers don't need to be in quotes, but it will work either way.

Reply:
I did remove the quotes from numbers and booleans, and I noticed an error in the twenty-films-of-06